lib.ts

```diff
@@ -218,6 +218,10 @@ export async function getDestination(platform: string) {
       return ['-destination', `id=${id}`]
     case 'macOS':
       return ['-destination', 'platform=macOS']
+    case 'mac-catalyst':
+      return ['-destination', "platform=macOS,variant='Mac Catalyst'"]
+    case 'driver-kit':
+      return ['-destination', "platform=macOS,variant='DriverKit'"]
```

Comment:
I tried building CombineCloudKit for `mac-catalyst` with `swift: ~5.5` and `runs-on: macos-11`:
https://github.com/chris-araman/CombineCloudKit/pull/4/checks?check_run_id=2866696038

Looks like we need to use quote the arguments differently:
```
'platform=macOS,variant=Mac Catalyst'
'platform=macOS,variant=DriverKit'
```

I verified the quote changes (before and after) on my Mac, though not through GHA.